tests/testStatsdClient.cpp

```diff
@@ -112,9 +123,9 @@ void testSendRecv(uint64_t batchSize) {
         expected.emplace_back("sendRecv.myTiming:2|ms|@0.10");
 
         // Send a metric explicitly
-        client.send("tutu", 4, "c", 2.0f);
+        client.send("tutu", 241, "c", 2.0f);
```

Comment:
this was just because i thought it would be useful to exorcise some bigger numbers some times